src/main/java/com/example/springbootcalculator/controllers/Test.java

```diff
@@ -0,0 +1,48 @@
+package com.example.springbootcalculator.controllers;
+
+import com.example.springbootcalculator.services.Impl.CalculateServiceImpl;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+public class Test {
+    private final CalculateServiceImpl calculateService;
+
+    public Test(CalculateServiceImpl calculateService) {
+        this.calculateService = calculateService;
+    }
+
+    @GetMapping("/")
+    public String form() {
+        return "calculate";
+    }
+
+    @PostMapping("/calculate")
+    public String Calculate (@RequestParam("a") int a, @RequestParam("b") int b,
```

Comment:
It's better to create DTO with this 3 parameters and pass it as an argument to the method